github-react-app/src/components/RepoBranchesDrawer/RepoBranchesDrawer.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+
+import GitHubStore from "@store/GitHubStore";
+import { RepoBranches, RepoItem } from "@store/GitHubStore/types";
+import { Drawer } from "antd";
+import { List } from "antd";
+
+type RepoBranchesDrawerProps = {
+    selectedRepo: RepoItem | null;
+    onClose?: () => void;
+    visible: boolean;
+};
+
+const RepoBranchesDrawer: React.FC<RepoBranchesDrawerProps> = ({
+    selectedRepo,
+    onClose,
+    visible,
+}) => {
+    const [branches, setBranches] = useState<RepoBranches[] | null>(null);
+
+    useEffect(() => {
+        const func = async () => {
+            const gitStore = new GitHubStore();
+            if (selectedRepo !== null) {
+                await gitStore
+                    .getOrganizationRepoBranchesList({
+                        organizationName: selectedRepo?.orgName,
+                        repoName: selectedRepo.repoName,
+                    })
+                    .then((result) => setBranches(result.data));
```

Comment:
не совсем корректно используете `async/await`
```suggestion
                const result = await gitStore
                    .getOrganizationRepoBranchesList({
                        organizationName: selectedRepo?.orgName,
                        repoName: selectedRepo.repoName,
                    });
                    setBranches(result.success ? result.data : []);
```